public/robots.txt

```diff
@@ -1,5 +1,11 @@
 # See http://www.robotstxt.org/robotstxt.html for documentation on how to use the robots.txt file
 #
 # To ban all spiders from the entire site uncomment the next two lines:
-# User-agent: *
-# Disallow: /
+User-agent: *
+Disallow: /
```

Comment:
You shouldn't be doing this I fear. It's disallowing *all* paths (so the following lines would be irrelevant then 😉)